src/servises/formatString.ts

```diff
@@ -0,0 +1,3 @@
+export const formatString = (str: string): string => {
+  return str.toLowerCase().split(' ').join(' ');
```

Comment:
The current implementation of `formatString` does not modify spaces as it splits and joins the string with the same delimiter (' '). If the intention is to normalize spaces, consider using a regular expression to replace multiple spaces with a single space.